examples/spring-sdk/src/main/java/io/honeycomb/examples/springbootsdk/Application.java

```diff
@@ -0,0 +1,48 @@
+package io.honeycomb.examples.springbootsdk;
+
+import java.util.Arrays;
+
+import io.honeycomb.opentelemetry.HoneycombSdk;
+import io.honeycomb.opentelemetry.sdk.trace.spanprocessors.BaggageSpanProcessor;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class Application {
+
+	@Bean
+	public HoneycombSdk honeycomb() {
+			return new HoneycombSdk.Builder()
+					.addSpanProcessor(new BaggageSpanProcessor())
+					.setApiKey(System.getenv("HONEYCOMB_API_KEY"))
+					.setDataset(System.getenv("HONEYCOMB_DATASET"))
+					// .setServiceName(System.getenv("SERVICE_NAME"))
```

Comment:
We should remove the commented out version of `setServiceName`

Reply:
Good catch!